samples/debug/ppi_trace/Kconfig

```diff
@@ -0,0 +1,67 @@
+# Kconfig - Config options for PPI trace sample
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+mainmenu "PPI trace sample"
+
+menu "PPI trace pins configuration"
+
+config PPI_TRACE_PIN_RTC_COMPARE_EVT
+	int "Trace pin for RTC Compare event"
+	default 17 if BOARD_NRF9160_PCA10090
+	default 1 if BOARD_NRF51_PCA10028
```

Comment:
Currently you cannot build the `ppi_trace` module on nRF51. This entry (and similar ones below) may falsely suggest the opposite.

Reply:
fixed.